lib/registries/addon/components/comment-card/template.hbs

```diff
@@ -0,0 +1,114 @@
+{{#if this.loading}}
+    <LoadingIndicator @dark={{true}} />
+{{else}}
+    {{#if this.isAbuse}}
+        <div local-class="CommentRestore">
+            <span>{{t "registries.overview.comments.comment_abuse"}}</span>
+            {{#if this.currentUserHasReported}}
+                <a role="button" onclick={{perform this.submitRetractReport}}>
+                    {{t "registries.overview.comments.comment_abuse_action"}}
+                </a>
+            {{/if}}
+        </div>
+    {{else}}
+        <div local-class="Comment">
+            <div local-class="CommentHeader">
+                <div>
+                    {{#if this.comment.user.profileImage}}
+                        <img data-test-gravatar alt="{{t 'auth_dropdown.user_gravatar'}}" src={{concat this.comment.user.profileImage "&s=30"}}>
+                    {{/if}}
+                    <a role="link" href={{this.comment.user.profileURL}} local-class="CommentUser">
+                        {{this.comment.user.fullName}}
+                    </a>
+                </div>
+                <div>
+                    {{#if this.comment.modified}}
+                        <span local-class="dateModified">
+                            {{#bs-tooltip local-class="Tooltip" placement="left" viewportSelector=(concat "#" this.elementId)}}
+                                {{t 'registries.overview.comments.modified'}}{{this.dateModified}}
+                            {{/bs-tooltip}}
+                            <sup>{{fa-icon 'asterisk'}}</sup>
+                        </span>
+                    {{/if}}
+                    <span local-class="dateCreated">
+                        {{this.dateCreated}}
+                    </span>
+                </div>
+            </div>
+            <div local-class="CommentContent">
```

Comment:
It might make sense for `CommentReport` to be a sibling to `CommentContent`, not a child. So a structure like:
```
div.Comment:
  div.CommentHeader
  div.CommentContent
  if reportMode:
    form.CommentReport
  else:
    div.CommentControls
  if showReplies:
    div.CommentReplies
```